src/routes/lnurl/+page.svelte

```diff
@@ -0,0 +1,95 @@
+<script lang="ts">
+  import { decodelnurl } from 'js-lnurl'
+  import { translate } from '$lib/i18n/translations'
+  import Spinner from '$lib/elements/Spinner.svelte'
+  import Auth from './components/auth.svelte'
+  import type { PageData } from './$types'
+  import Pay from './components/pay.svelte'
+  import { LNURL_PROXY } from '$lib/constants'
+  import warning from '$lib/icons/warning'
+
+  export let data: PageData
+
+  let parsingLnurl = false
+  let parseLnurlError = ''
+
+  let url: URL
+  let tag: string
+  let k1: string
+  let action: string
+  let callback: string // The URL from LN SERVICE which will accept the pay request parameters
+  let maxSendable: number // Max millisatoshi amount LN SERVICE is willing to receive
+  let minSendable: number // Min millisatoshi amount LN SERVICE is willing to receive, can not be less than 1 or more than `maxSendable`
+  let metadata: string // Metadata json which must be presented as raw string here, this is required to pass signature verification at a later step
+  let commentAllowed: number // length of comment allowed
```

Comment:
maybe better named `commentLengthAllowed`?

Reply:
This comes straight from the spec and I think it makes sense since a value of `0` indicates no comment is allowed, and then any positive value indicates it is allowed and the length that is allowed.